src/main/js/main_html5.js

```diff
@@ -1483,8 +1483,21 @@ require("../../../html5-common/js/utils/environment.js");
         this.controller.notify(this.controller.EVENTS.PAUSED);
       }
       forceEndOnPausedIfRequired();
+      removeControlsAttr();
```

Comment:
Does anything happen when the video is paused in fullscreen if we remove the controls attribute when pausing?

Reply:
No, nothing happen when the video is paused in fullscreen, but i put the function to handler for "webkitendfullscreen" (according JP's idea).